backend/compact-connect/lambdas/license-data/handlers/providers.py

```diff
@@ -87,6 +89,12 @@ def query_providers(event: dict, context: LambdaContext):  # pylint: disable=unu
 
 @api_handler
 def get_provider(event: dict, context: LambdaContext):  # pylint: disable=unused-argument
+    """
+    Return one provider's data
+    :param event: Standard API Gateway event, API schema documented in the CDK ApiStack
+    :param LambdaContext context:
+    :return:
```

Comment:
Does this have significance?

Reply:
The `:return:`? It's just automatically added to the bottom of doc-strings by most python IDEs when you start adding parameters, etc. An empty `:return:` line has no meaningful effect, really.

Reply:
Nit but mind cutting or otherwise documenting the return value?